src/Form/Extension/OrderTypeExtension.php

```diff
@@ -26,15 +27,21 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 ->add('items', OrderItemCollectionType::class, [
                     'entry_options' => ['currency_code' => $order->getCurrencyCode()],
                 ])
+                ->add('discounts', OrderDiscountCollectionType::class, [
+                    'property_path' => 'adjustments',
+                    'entry_options' => [
+                        'currency' => $order->getCurrencyCode(),
+                    ],
+                    'button_add_label' => 'setono_sylius_order_edit.ui.add_discount',
+                ])
+                ->add('storeNotes', TextareaType::class, [
+                    'required' => false,
+                    'label' => 'setono_sylius_order_edit.ui.store_notes',
+                    'attr' => [
+                        'placeholder' => 'setono_sylius_order_edit.ui.store_notes_placeholder',
+                    ],
+                ])
```

Comment:
I believe this field could be added outside of event listener, as it does not require any data from the form data (as the two previous need)